doc/tutorials/content/conf.py

```diff
@@ -19,17 +17,17 @@
 master_doc = 'index'
 
 # General information about the project.
-project = u'PCL'
+project = u'Point Cloud Library'
 copyright = ''
 
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 #
 # The short X.Y version.
-version = '0.0'
+version = '1.10.1-dev'
```

Comment:
Is this actually necessary given that you pass `-Dversion="${PCL_VERSION_PRETTY}"` in `add_custom_target`?

Reply:
Not at all. Even RTD overrides it in its own build.